### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by choenes** [shama/grunt-docs#18]
I am dealing with the dependency on docpad and ran into the following:
peerinvalid The package docpad does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer grunt-docs@0.8.0 wants docpad@~6.64.3
npm ERR! peerinvalid Peer docpad-plugin-marked@2.2.1 wants docpad@6
npm ERR! peerinvalid Peer docpad-plugin-eco@2.1.0 wants docpad@6
npm ERR! peerinvalid Peer docpad-plugin-highlightjs@2.2.2 wants docpad@6

Do those plugins have to be upgraded or be looser about which version they accept?

**Comment by shama** [comment]
Yeah we probably should do `"docpad": "^6.64.3"` with docpad or just remove the peer dep all together (as they are going away in future versions of npm anyways).
